internal/md2man/md2man.go

```diff
@@ -0,0 +1,20 @@
+package md2man
```

Comment:
This isn't a super strong opinion, but could we put this underneath `docs/internal` or something? I don't expect we'd use this as part of `av` itself.[](url)

Reply:
Done.